eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -179,6 +189,7 @@ private Collection<Dependency> deps() throws IOException {
             deps.add(one);
             tojo.set(AssembleMojo.ATTR_JAR, coords);
         }
+        new Conflicted(deps).check(!this.ignVerConflicts);
```

Comment:
@mximp it's not a good practice in general: to make objects, which in reality are just procedures that look like objects. in this case, you can simply do the checking in a private static method.